sqlmesh/core/model/kind.py

```diff
@@ -120,6 +120,10 @@ def model_kind_name(self) -> t.Optional[ModelKindName]:
     def to_expression(self, **kwargs: t.Any) -> d.ModelKind:
         return d.ModelKind(this=self.name.value.upper(), **kwargs)
 
+    @property
+    def data_hash_fields(self) -> t.List[str]:
```

Comment:
Consider renaming to something like `data_hash_values` since it returns the values and not the field names.

Reply:
I actually followed the convention from `model/definition.py`. Should I still rename this?

Reply:
In my opinion, the name of the method is misleading. If you agree, change it; otherwise, fine as is.

Reply:
renamed